math.gobra

```diff
@@ -0,0 +1,62 @@
+// Package math provides common math functions for verification
+package math
+
+// Max returns the maximum of x and y
```

Comment:
Let's drop the trivial comments for `Min`, `Max` (and its variants with 3 elements), and `Abs`